Description: get_action_area() now returns ButtonBox* instead of 
 HButtonBox*, change type of constructor argument to reflect this.
Author: Ilya Barygin <barygin@gmail.com>
--- workrave-1.9.1.orig/frontend/plugin/exercises/gtkmm/src/ExercisesPanel.hh
+++ workrave-1.9.1/frontend/plugin/exercises/gtkmm/src/ExercisesPanel.hh
@@ -43,7 +43,7 @@
 class ExercisesPanel : public Gtk::HBox
 {
 public:
-  ExercisesPanel(Gtk::HButtonBox *dialog_action_area);
+  ExercisesPanel(Gtk::ButtonBox *dialog_action_area);
   ~ExercisesPanel();
 
   void set_exercise_count(int num);
--- workrave-1.9.1.orig/frontend/plugin/exercises/gtkmm/src/ExercisesPanel.cc
+++ workrave-1.9.1/frontend/plugin/exercises/gtkmm/src/ExercisesPanel.cc
@@ -210,7 +210,7 @@ text_buffer_set_markup (GtkTextBuffer *b
 
 int ExercisesPanel::exercises_pointer = 0;
 
-ExercisesPanel::ExercisesPanel(Gtk::HButtonBox *dialog_action_area)
+ExercisesPanel::ExercisesPanel(Gtk::ButtonBox *dialog_action_area)
   : Gtk::HBox(false, 6),
          exercises(Exercise::get_exercises())
 {
